src/sql/src/names.rs

```diff
@@ -723,6 +734,16 @@ pub enum DatabaseId {
     System(u64),
 }
 
+impl DatabaseId {
+    pub fn is_user(&self) -> bool {
```

Comment:
unused?

Reply:
This is unused, but it's fairly small and I think since we're adding `is_system` it's useful to have this around for the future.